In [17]:
queries = [
    'смотреть сериалы онлайн',
    'новости спорта',
    'афиша кино',
    'курс доллара',
    'сериалы этим летом',
    'курс по питону',
    'сериалы про спорт',
]
total = len(queries)    #количество элементов списка
word_count = {}    #пустой словарь
for i in queries:
    if len(i.split()) in word_count:    
        word_count[len(i.split())] += 1  #добавление в пустой словарь по ключу на каждое следуещее по счету слово
    else:
         word_count[len(i.split())] = 1  
for N in word_count.values():
    percentage = 100 * (N / total)
    print('Поисковых запросов, содержащих 2 слова:', round(percentage, 2), '%')

Поисковых запросов, содержащих 2 слова: 57.14 %
Поисковых запросов, содержащих 2 слова: 42.86 %


Дана переменная, в которой хранится список поисковых запросов пользователя (пример структуры данных приведен ниже, но запросы потенциально могут быть любые). Вам необходимо написать программу, которая выведет на экран распределение количества слов в запросах в требуемом виде.

Пример работы программы:

queries = [
    'смотреть сериалы онлайн',
    'новости спорта',
    'афиша кино',
    'курс доллара',
    'сериалы этим летом',
    'курс по питону',
    'сериалы про спорт',
]
Результат:

Поисковых запросов, содержащих 2 слов(а): 42.86%
Поисковых запросов, содержащих 3 слов(а): 57.14%